oracle-jdk-installer/linux/11/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
```

Comment:
JFYI: Alpine may have performance issue probably derived from `musl`. https://superuser.com/questions/1219609/why-is-the-alpine-docker-image-over-50-slower-than-the-ubuntu-image

Reply:
I see, thanks for raising this.
If we end up implementing this this [changes](https://github.com/scalar-labs/scalardb-cluster/pull/592#discussion_r1669826570), I will need to run the ScalarDB cluster integration test from the image so I will consider Alpine running performance issue. 
But if we only use the image to extract the JDK installer, the running performance of Alpine is not an issue since the container is never run, as I explain in the first paragraph of the [README](https://github.com/scalar-labs/docker/blob/326e56e63b84543eab95c840ae29b0bd7aed839c/oracle-jdk-installer/README.md#docker-images-containing-installer-for-oracle-jdk-8-and-11-on-linux-and-windows) of this PR

Reply:
Ah, sorry. I missed the usage. It makes sense.